README.md

```diff
@@ -21,24 +21,23 @@ alignmentUrl   |            |             | schema:targetUrl
-Endorsed            | TBA       | Issued an endorsement or annotation of an Issuer, BadgeClass, or Assertion that should be published
-Displayed           | TBA       | Displayed badge details on a single badge to a consumer
-Accepted            | TBA       | Inspected a badge or pathway with completion data and accepted it in exchange for some privilege, opportunity, access, or other benefit granted to the recipient.
-Applied             | TBA       | User submits evidence to an issuer in hopes of earning a badge.
-AssignedToObjective | TBA       | Set BadgeClass as related to prospective opportunity, privilege, objective, learning pathway, skill or competency definition
+Action | IRI | Description | New   |
+-------|-----|------------ | :---: |
+Defined | http://purl.imsglobal.org/vocab/caliper/action#Defined | Defined a new BadgeClass or Issuer | Yes |
+Issued  | http://purl.imsglobal.org/vocab/caliper/action#Issued | Awarded an instance of a BadgeClass to a recipient, perhaps accepting an application | Yes
+MetCriteria | http://purl.imsglobal.org/vocab/caliper/action#MetCriteria | Recipient has met criteria for a BadgeClass and should be awarded a badge. | Yes
+Modified | http://purl.imsglobal.org/vocab/caliper/action#Modified | Modified or updated a previously defined badge object with new metadata | No
+Revoked | http://purl.imsglobal.org/vocab/caliper/action#Revoked | Updated a previously awarded badge to mark it as revoked | Yes
+Deleted | http://purl.imsglobal.org/vocab/caliper/action#Deleted | Marked a defined badge object as deleted | No
+Registered | http://purl.imsglobal.org/vocab/caliper/action#Registered | Registered a recipient profile with a backpack endpoint or issuer profile with issuer endpoint | Yes
+Requested | http://purl.imsglobal.org/vocab/caliper/action#Requested | Requested a badge object or related resource from an issuer or repository | Yes
+Verified | http://purl.imsglobal.org/vocab/caliper/action#Verified | Verified an Open Badge's authenticity and structural integrity | Yes
+Shared | http://purl.imsglobal.org/vocab/caliper/action#Shared | Triggered or authorized a share of a single badge, collection, or other document including badges | No
+Endorsed | http://purl.imsglobal.org/vocab/caliper/action#Endorsed | Issued an endorsement or annotation of an Issuer, BadgeClass, or Assertion that should be published | Yes
+Displayed | http://purl.imsglobal.org/vocab/caliper/action#Displayed | Displayed badge details on a single badge to a consumer | Yes
+Accepted | http://purl.imsglobal.org/vocab/caliper/action#Accepted | Inspected a badge or pathway with completion data and accepted it in exchange for some privilege, opportunity, access, or other benefit granted to the recipient. | Yes
+Applied | http://purl.imsglobal.org/vocab/caliper/action#Applied | User submits evidence to an issuer in hopes of earning a badge. | Yes
```

Comment:
The rest of these are in past tense. What is the common pattern for tense? 
* user submits evidence
* submit evidence
* user submitted evidence